devices/wiser/fuga_double_relay_switch.json

```
@@ -1,211 +1,205 @@
-        "0x02"
-      ],
-      "items": [
-        {
-          "name": "attr/id"
-        },
-        {
-          "name": "attr/lastannounced"
-        },
-        {
-          "name": "attr/lastseen"
-        },
-        {
-          "name": "attr/manufacturername"
-        },
-        {
-          "name": "attr/modelid"
-        },
-        {
-          "name": "attr/name"
-        },
-        {
-          "name": "attr/swversion"
-        },
-        {
-          "name": "attr/type"
-        },
-        {
-          "name": "attr/uniqueid"
-        },
-        {
-          "name": "state/alert",
-          "default": "none"
-        },
-        {
-          "name": "state/on",
-          "refresh.interval": 5
-        },
-        {
-          "name": "state/reachable"
-        }
-      ]
-    },
-    {
-      "type": "$TYPE_SWITCH",
-      "restapi": "/sensors",
-      "uuid": [
-        "$address.ext",
-        "0x15",
-        "0x0006"
-      ],
-      "items": [
-        {
-          "name": "attr/id"
-        },
-        {
-          "name": "attr/lastannounced"
-        },
-        {
-          "name": "attr/lastseen"
-        },
-        {
-          "name": "attr/manufacturername"
-        },
-        {
-          "name": "attr/modelid"
-        },
-        {
-          "name": "attr/name"
-        },
-        {
-          "name": "attr/swversion"
-        },
-        {
-          "name": "attr/type"
-        },
-        {
-          "name": "attr/uniqueid"
-        },
-        {
-          "name": "config/on"
-        },
-        {
-          "name": "config/reachable"
-        },
-        {
-          "name": "state/buttonevent",
-          "parse": {
-            "cl": "0x0006",
-            "ep": 255,
-            "eval": "if (SrcEp >= 21 && SrcEp <= 24) { Item.val = (SrcEp - 20) * 1000 + 2; }",
-            "fn": "zcl"
-          }
-        },
-        {
-          "name": "state/lastupdated"
-        }
-      ]
-    }
-  ],
-  "bindings": [
-    {
-      "bind": "unicast",
-      "src.ep": 21,
-      "cl": "0x0006"
-    },
-    {
-      "bind": "unicast",
-      "src.ep": 22,
-      "cl": "0x0006"
-    },
-    {
-      "bind": "unicast",
-      "src.ep": 23,
-      "cl": "0x0006"
-    },
-    {
-      "bind": "unicast",
-      "src.ep": 24,
-      "cl": "0x0006"
-    },
-    {
-      "bind": "unicast",
-      "src.ep": 1,
-      "cl": "0x0006",
-      "report": [
-        {
-          "at": "0x0000",
-          "dt": "0x10",
-          "min": 1,
-          "max": 600
-        }
-      ]
-    },
-    {
-      "bind": "unicast",
-      "src.ep": 2,
-      "cl": "0x0006",
-      "report": [
-        {
-          "at": "0x0000",
-          "dt": "0x10",
-          "min": 1,
-          "max": 600
-        }
-      ]
-    }
-  ]
-}
\ No newline at end of file
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "Schneider Electric",
+  "modelid": "LK Switch",
+  "product": "LK Fuga Doube Relay Switch",
```

Perhaps this typo could be fixed?
```suggestion
  "product": "LK Fuga Double Relay Switch",
```